pstatus-graphql-ktor/README.md

```diff
@@ -1,8 +1,34 @@
 # Overview
-This project is the processing status report sink.  It listens for messages on an Azure Service bus, validates the messages, and if validated persists them to CosmosDB.
+This project is the processing status service. The pstatus-graph-ktor service provides the following features:
```

Comment:
" graphQL service " missing in the first sentence?

Reply:
Corrected the description.